localtypings/pxtarget.d.ts

```diff
@@ -177,6 +177,7 @@ declare namespace pxt {
         trustedUrls?: string[]; // URLs that are allowed in simulator modal messages
         invalidatedClass?: string; // CSS class to be applied to the sim iFrame when it needs to be updated (defaults to sepia filter)
         stoppedClass?: string; // CSS class to be applied to the sim iFrame when it isn't running (defaults to grayscale filter)
+        keymap?: boolean; // when non-empty and autoRun is disabled, this code is run upon simulator first start
```

Comment:
Real minor nitpick, the comment here doesn't really indicate what the flag does to me - looks like it might be takem from elsewhere?